kitchen.dokken.yml

```diff
@@ -1,7 +1,7 @@
 driver:
   name: dokken
   privileged: true # because Docker and SystemD/Upstart
-  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
```

Comment:
Was there a reason to change this. This gets us early failure detection on new chef releases, which is quite nice from a quality standpoint.

Reply:
previous travis builds failed to compile runit dependency cookbook. I suspected it was due to chef-client 15.0.143 that it was using.

Reply:
looks like I was wrong as it's still failing at the same spot. Hopefully @josh-barker may be able to point out the issue with the tests.

Reply:
runit is entirely busted at the moment (new release today maybe) in Chef 15 so this does need to be stable at the moment.